trajopt/include/trajopt/problem_description.hpp

```diff
@@ -259,6 +259,8 @@ struct CartPoseTermInfo : public TermInfo
   std::string link;
   /** @brief Static transform applied to the link */
   Eigen::Isometry3d tcp;
+  /** @brief The frame relative to which the target position is defined */
+  std::string target;
```

Comment:
Might add a comment that if empty string the frame is assumed to be in world coordinate system.